### PR TITLE
docs: Small typos in libraries/uploaded_files.rst

### PR DESCRIPTION
**Pull request by obozdag**
**Description**
Small typos in libraries/uploaded_files.rst

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/libraries/uploaded_files.rst [hunk 1]
```diff
@@ -398,8 +398,4 @@ passing it as the second parameter:
 
 .. literalinclude:: uploaded_files/020.php
 
-Moving an uploaded file can fail, with an ``HTTPException``, under several circumstances:
-
-- the file has already been moved
-- the file did not upload successfully
-- the file move operation fails (e.g., improper permissions)
```
Comment (kenjis) [inline user_guide_src/source/libraries/uploaded_files.rst:405]: Don't delete the list. We don't expect users to read another section.

File: user_guide_src/source/libraries/uploaded_files.rst [hunk 2]
```diff
@@ -83,9 +83,9 @@ this code and save it to your **app/Controllers** directory:
 
 .. literalinclude:: uploaded_files/002.php
 
-Since the value of a file upload HTML field doesn't exist, and is stored in the
+Since HTML file element has no value attribute, and the upload file is stored in the
```
Comment (kenjis) [inline user_guide_src/source/libraries/uploaded_files.rst:86]: The original sentence is also difficult to understand, but this also seems inaccurate.
The original says there is no data for uploaded files in `$_POST`.

I don't understand what "HTML file element has no value attribute" says.

File: user_guide_src/source/libraries/uploaded_files.rst [hunk 3]
```diff
@@ -83,9 +83,9 @@ this code and save it to your **app/Controllers** directory:
...
 ``$_FILES`` global, only :ref:`rules-for-file-uploads` can be used to validate
```
Comment (kenjis) [inline user_guide_src/source/libraries/uploaded_files.rst:87]: "the upload file is stored in the ``$_FILES`` global"
This is not true or misleading. 
The global [$_FILES](https://www.php.net/manual/en/reserved.variables.files.php) contains all the uploaded file information.

**Review by kenjis — APPROVED** [review]

File: user_guide_src/source/libraries/uploaded_files.rst (hunk 2 above)
Comment (kenjis) [reply user_guide_src/source/libraries/uploaded_files.rst]: I sent another PR to improve the description. See #8910

**Review by datamweb — APPROVED** [review]
Thanks!

**Pull request merged**